lib/Net/DRI/DRD/NGTLD.pm

```diff
@@ -1037,16 +1037,18 @@ L<Net::DRI::Protocol::EPP::Extensions::TangoRS::Auction> urn:ar:params:xml:ns:au
 
 L<Net::DRI::Protocol::EPP::Extensions::TangoRS::LaunchPhase> : http://xmlns.corenic.net/epp/mark-ext-1.0
 
-L<Net::DRI::Protocol::EPP::Extensions::CentralNic::Fee> urn:centralnic:params:xml:ns:fee-0.6
-
-Fee extension is currently only used in .NRW and for domain_check command only.
-
 =cut
 
+ my $tangorsversion; # setting this variable because this profile use distinct fee extensions per TLD :(
+ $tangorsversion = undef if $tld eq 'ruhr'; # no fee extension
```

Comment:
Should this not be default? I.e. set to undef for all, not just for 'ruhr', and then only set for specific TLDs? In this case, gmx wouldn't be set to anything ...